config/Dockerfiles/cloud-image-compose/virt-customize.sh

```diff
@@ -80,7 +80,7 @@ EOF
 
 virt-copy-in -a ${DOWNLOADED_IMAGE_LOCATION} ${CURRENTDIR}/testrepo/${package} ${CURRENTDIR}/test-${package}.repo /etc/yum.repos.d/
 
-for pkg in $(repoquery --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --all | grep -v '\-debug\|\-devel\|.src' | rev | cut -d '-' -f 3- | rev ) ; do
+for pkg in $(repoquery -q --disablerepo=\* --enablerepo=${package} --repofrompath=${package},${rpm_repo} --all | grep -v '\-debuginfo$\|\-debugsource$\|\.src$' | sed 's|-[^-]*-[^-]*$||') ; do
```

Comment:
actually I just realized that first you need to get the name and only after that grep for debuginfo/debugsource